src/index.json

```diff
@@ -812,7 +812,7 @@
                     "metadata_version": "2.0",
                     "name": "botservice",
                     "summary": "Bug fixes for issues in the native botservice cli command module.",
```

Comment:
Shouldn't this summary also include that this extension is used for previews?

Reply:
so far, we havent included any features that are preview only in this extension. most of the commits go to the native cli. We are making an update to the extension in this case for botbuilder v4 because we were coming in too hot to get onto the native cli release timeline